mkdocs.yml

```diff
@@ -12,7 +12,6 @@ pages:
 - ['getting_started/flow_control.md', 'Creating applications on mbed OS', 'Application flow control and task management']
 - ['dev_tools/options.md', 'Development tools and aids', 'Development tool options']
 - ['dev_tools/cli.md', 'Development tools and aids', 'mbed CLI']
-- ['dev_tools/cli_install.md', 'Development tools and aids', 'mbed CLI for Windows installer']
```

Comment:
If we remove this from the YAML, your link on the other PR will not go anywhere because this page won't exist.